scripts/postinstall.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+if [ -f templates.tgz ]; 
+  then 
+  tar xf templates.tgz
+fi
+
+echo """
```

Comment:
The message is a little big huge I think, can we make it on only one line?

Reply:
Updated